### DISCUSSION TITLE
How to restore and retrain with different input parameter

### DISCUSSION THREAD

**Discussion by LeeStr1** [lululxvi/deepxde#1908] [Q&A]
I want to train model with low Rayleigh number first for natural convection. and then restore this model to retrain for higher Rayleigh number. I tried following the strategy by reading previous discussions on this forum. But, in my case, it's not working and I am getting errors. I want to know what is the standard method to do this properly.

**Comment by vl-dud** [comment]
You can use `model_restore_path` argument to restore the model for retraining:
```
model = dde.Model(pde, net)
model.compile("adam", lr=0.001)
model.train(
  iterations=10000,
  display_every=1000,
  model_restore_path="path_to_saved_model"
)
```

**Reply by LeeStr1** [reply]
I was able to solve the error by naming model2 instead of model in retraining.
